docs/deployment/cloud.rst

```diff
@@ -129,81 +127,79 @@ always will work is to use the ``port-forward`` command:
     * `AWS registration process <https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/>`_
-    * Azure is not supported in this process by now but we have plans to include it soon.
+    * Azure will be supported in a future release.
 
-Once time you have created an account, you will need to configure an API key/Access key to access from your terminal to your selected provider account:
+Once you have created an account, you will need to configure an API key/access key:
     * `IBM Cloud API key creation <https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui#create_user_key>`_
     * `AWS Access key creation <https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-about>`_
 
-And as last setup step install the provider's CLI:
+Finally, install the provider's CLI:
     * `IBM Cloud CLI <https://cloud.ibm.com/docs/cli?topic=cli-getting-started>`_
     * `AWS CLI <https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html>`_
 
-Now you have all the configuration needed from your cloud provider. The next step will be configure the terraform to
-deploy the infrastructure where you want.
+Now that your cloud provider is configured, you may configure ``terraform`` to
+deploy the infrastructure where you want. Depending on the provider, you need to
+apply different configurations:
 
-For that, depending of the provider, you need apply different configurations:
-
-**For IBM Cloud** is the easiest one. Just go to ``./infrastructure/terraform/ibm`` and create the file ``terraform.tfvars``
-with the next content:
+The easiest option is to use **IBM Cloud**. Just go to ``./infrastructure/terraform/ibm`` and create the file ``terraform.tfvars``
+with the following contents:
 
 .. code-block::
 
         ibmcloud_api_key = "YOUR_API_KEY"
         ibm_region = "YOUR_REGION"
         resource_group = "YOUR_RESOURCE_GROUP"
 
-* **Note**: check in the next links to know the `region <https://cloud.ibm.com/docs/openwhisk?topic=openwhisk-cloudfunctions_regions>`_ and the `resource group <https://cloud.ibm.com/docs/account?topic=account-rgs&interface=cli>`_ that your account have configured.
+* Note the `region <https://cloud.ibm.com/docs/openwhisk?topic=openwhisk-cloudfunctions_regions>`_ and the `resource group <https://cloud.ibm.com/docs/account?topic=account-rgs&interface=cli>`_ associated with your account.
 
-**In AWS** case instead to create a file you will need to configure a set of environment variables in your terminal as it
-is defined `here <https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html#envvars-set>`_.
+To use **AWS**, configure environment variables, depending on your system, in accordance with the instructions `here <https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html#envvars-set>`_.
 
-To confirm the configuration just run terraform:
+To confirm the configuration just run ``terraform``:
```

Comment:
one final nit: in the text we say just run `terraform`, but the code block shows running `terraform plan` ...